tools/topology/sof-tgl-rt711-rt1308.m4

```diff
@@ -56,20 +74,6 @@ PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
 	1000, 0, 0,
 	48000, 48000, 48000)
 
-# Low Latency playback pipeline 3 on PCM 2 using max 2 channels of s32le.
-# Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
-	3, 2, 2, s32le,
-	1000, 0, 0,
-	48000, 48000, 48000)
```

Comment:
was this intentional @plbossart or did you mean to remove pipeline 5?

Reply:
nope, that's probably the issue. thanks!